test/fastmath.jl

```diff
@@ -202,3 +202,10 @@ let a = ones(2,2), b = ones(2,2)
     local c = 0
     @test @fastmath(c |= 1) == 1
 end
+
+struct LitPowTest end
+Base.literal_pow(::typeof(Base.FastMath.pow_fast), ::LitPowTest, ::Val{p}) where {p} = 1
```

Comment:
does this mean if library authors want to use `^` with `@fastmath` they also have to add both
```
Base.literal_pow(::typeof(Base.FastMath.pow_fast),...
Base.literal_pow(::typeof(^),...
```